src/next/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,331 @@
+            alignment === "left"
+              ? "md:justify-start md:text-start"
+              : "md:justify-end md:text-end"
+          }`}
+        >
+          <div
+            className={`p-8 flex flex-col gap-6 ${bgColor} ${textColor} md:w-2/3 lg:w-1/2`}
+          >
+            <h1 className="text-6xl font-medium">{title}</h1>
+            {subtitle && <p>{subtitle}</p>}
+            {buttonLabel && buttonUrl && (
+              <div
+                className={`flex flex-row gap-4 ${
+                  alignment === "left"
+                    ? "justify-center md:justify-start"
+                    : "justify-center md:justify-end"
+                }`}
+              >
+                <Button
+                  colorVariant="white"
+                  label={buttonLabel}
+                  href={buttonUrl}
+                />
+                {secondaryButtonLabel && secondaryButtonUrl && (
+                  <Button
+                    colorVariant="black"
+                    label={secondaryButtonLabel}
+                    href={secondaryButtonUrl}
+                  />
+                )}
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    </section>
+  )
+}
+
+const HeroCopyLed = ({
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: Omit<HeroCopyLedProps, "variant">) => {
+  return (
+    <section className="h-[32rem]">
+      <div className="bg-[#ffeec2] w-full flex flex-row justify-center">
+        <div className="w-4/5 lg:w-3/4 py-16 flex flex-col items-center text-center md:items-start md:text-start gap-6">
+          <h1 className="text-6xl font-medium">{title}</h1>
+          {subtitle && <p>{subtitle}</p>}
+          {buttonLabel && buttonUrl && (
+            <div className="flex flex-row gap-4">
+              <Button
+                colorVariant="white"
+                label={buttonLabel}
+                href={buttonUrl}
+              />
+              {secondaryButtonLabel && secondaryButtonUrl && (
+                <Button
+                  colorVariant="black"
+                  label={secondaryButtonLabel}
+                  href={secondaryButtonUrl}
+                />
+              )}
+            </div>
+          )}
+        </div>
+      </div>
+      {backgroundUrl && (
+        <div
+          className="bg-cover bg-center bg-no-repeat h-3/5"
+          style={{
+            backgroundImage: `url('${backgroundUrl}')`,
+          }}
+        ></div>
+      )}
+    </section>
+  )
+}
+
+const HeroFloatingImage = ({
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: HeroFloatingImageProps) => {
+  return (
+    <>
+      <section className="hidden md:block h-[32rem]">
+        <div className="bg-[#ffeec2] w-full flex flex-row justify-center">
+          <div className="w-4/5 lg:w-3/4 py-16 flex flex-row gap-4">
```

Comment:
nit: might want to increase the `gap`  between text and image slightly 

<img width="933" alt="Screenshot 2024-03-09 at 2 08 54 AM" src="https://github.com/isomerpages/isomer-components/assets/77665182/ffb41fee-c692-4d89-947b-a55ed110ac0d">

Reply:
Adjusted in 8ee7e6894f8ec730fbe37e1b68663fca2b02db4c.